uaclient/tests/test_data_types.py

```diff
@@ -249,6 +366,22 @@ def __init__(
         {"string": "nestedstring2_opt", "integer": 66},
         {"string": "nestedstring3_opt", "integer": 77},
     ],
+    "enum_list": ["two", "three"],
+    "enum": "one",
+    "enum_opt_list": ["one", "four"],
```

Comment:
`example_data_object_dict_with_optionals` should include a value for all of the optional fields. So we need a value for `enum_opt` here.

Reply:
Yes, now the earlier comment makes sense.